### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/zh/guide/waline.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#584] [Announcements]
# https://plugin-comment2.vuejs.press/zh/guide/waline.html

安装 tab pnpm tab yarn tab npm 使用 tab TS tab JS 从 V1 迁移 详见 迁移指南 (./migration.md) import HighlightPanel from 

https://plugin-comment2.vuejs.press/zh/guide/waline.html

**Comment by lingdu990130556** [comment]
可以同时开启两种评论吗？比如我在全局使用Giscus，然后个别页面不是开发人员比如留言板啥的开启这个人人都能留言的功能
